docs/concepts/time-quantums.md

```diff
@@ -40,6 +40,40 @@ You can associate multiple times with each value, so a value only has to exist i
 
 Whenever a record with time quantums is ingested, a view is created for each level of granularity specified. This is essentially a copy of the column over a specific time range. If `YMDH` is specified and the time `2018-08-31T22:30:00Z` is ingested, a time view will exist for `2018`, `2018-08`, `2018-08-31`, and `2018-08-31T22`. This means data which has times for every hour for two days (say May 2nd and 3rd) in a column with `YMDH` time quantums configured will have 48+2+1+1+1 views (53) in total. 48 hours, 2 days, 1 month, 1 year, and the standard view.
 
+<!--
```

Comment:
I don't think this should go in here as it's specific to IDK, not time quantums

Reply:
I'll comment out for now with that note because the Time quantum stuff is going to be extensively reworked and I need to keep similar stuff in the same place so it's easier to find

Reply:
Commented out and committed to branch